docs/reference/glossary.asciidoc

```diff
@@ -95,27 +100,33 @@
   Each document is stored in a single primary <<glossary-shard,shard>>. When
   you index a document, it is indexed first on the primary shard, then
   on all <<glossary-replica-shard,replicas>> of the primary shard.
-  +
   By default, an <<glossary-index,index>> has 5 primary shards. You can
   specify fewer or more primary shards to scale the number of
   <<glossary-document,documents>> that your index can handle.
-  +
   You cannot change the number of primary shards in an index, once the
   index is created.
-  +
   See also <<glossary-routing,routing>>
 
+[[glossary-query]] query ::
+
+  A query is the basic component of a search. A search can be defined by one or more queries 
+  which can be mixed and matched in endless combinations. While <<glossary-filter,filters>> are
+  queries that only determine if a document matches, those queries that also calculate or score 
+  how well the document matches are known as "scoring queries". A scoring query calculates how 
```

Comment:
I'd scratch "calculates how well a document matches a query" here becauce this is already mentioned in the previous sentence. Maybe just use "Those queries assigns a score, which ..." and also leave out "... by score" at the end, too much repetition.